src/domain/Concept.ts

```diff
@@ -31,8 +31,8 @@ export interface Concept {
     forholdTilKilde: string;
     kilde: Kilde[];
   } | null;
-  merknad?: TekstMedSpraakKode;
```

Comment:
Liker ikke helt at man bruker Record for liste og egen interface for enkel verdi. Man mister litt poenget med typescript ved å bruke Record synes jeg. Men siden det er gjort andre steder også kan man diskutere om det lønner seg å bytte.

Reply:
Var slik det ble gjort på de andre feltene av den typen, tenkte det var greiest å fortsette med det. Kan heller se på en liten refaktorering av typene o.l. i repoet senere en gang.

Hvis jeg leste koden korrekt nå så er det forskjellige typer som brukes ved lagring vha import og vanlig lagring, som sannsynligvis er grunnen til at denne feilen oppstod. Bør uansett være noen som har bedre kontroll på typescript enn meg som ser på det :P